worlds/generic/docs/setup_en.md

```diff
@@ -47,39 +50,142 @@ slot name and, if the relevant game requires it, files to associate it with that
 If multiple people plan to play in one world cooperatively then they will only need one YAML for their coop world. If
 each player is planning on playing their own game then they will each need a YAML.
 
-### Gather All Player YAMLs
+### Generating a single player game
+
+#### On the website
+
+The easiest way to get started playing an Archipelago generated game, after following the base setup from the game's
+setup guide, is to find the game on the [Archipelago Games List](/games), click on `Settings Page`, set the settings how
+you want to play, and click `Generate Game` at the bottom of the page. This will create a page for the seed, from
+which you can create a room, and then [connect](#connecting).
+
+If you have downloaded the settings, or have created a settings file manually, this file can be uploaded on the
+[Generation Page](/generate) where you can also set any specific hosting settings.
+
+#### On your local installation
+
+To generate a game on your local machine, make sure to install the Archipelago software, and ensure to select the
+Generate component, as well as ROM setup, if necessary. Navigate to your Archipelago installation (usually 
+C:\ProgramData\Archipelago), and place the settings file you have either created or downloaded from the website in the
+`Players` folder.
+
+Run `ArchipelagoGenerate.exe`, and it will inform you whether the generation was successful or not. If successful, there
+will be an output zip in the `output` folder (usually named something like `AP_XXXXX.zip`). This will contain all
+relevant information to the session including the spoiler if one was generated.
+
+### Generating a multiplayer game
+
+Archipelago is a multi-game multiworld architecture, so any number of players and any number of games may be used to
+generate. Of note, the website currently limits to a maximum player count of 30. If you would like to generate a game
```

Comment:
"limits to a maximum" is a somewhat unusual construction, could use "has a maximum" instead